tasks/dep.py

```diff
@@ -28,12 +34,50 @@ def ensure_deps_declared(reqs_txt, reqs_in):
             f.writelines(declacred_lines)
 
 
+@task
+def dep_check(ctx):
+    packages = defaultdict(lambda: defaultdict(list))
```

Comment:
Would be nice to have some function documentation here and maybe a unit tests to be sure we cover all cases. What do you think?

Reply:
We don't have a way to test our tasks/tooling, currently.

This is an example of [Autovivification](https://en.wikipedia.org/wiki/Autovivification) and would be of default type `dict<dict<list>>`.